content/blog/20240430_fluentd-v1.17.0-has-been-released.md

```diff
@@ -0,0 +1,49 @@
+# Fluentd v1.17.0 has been released
+
+Hi users!
+
+We have released v1.17.0 on 2024-04-30. ChangeLog is [here](https://github.com/fluent/fluentd/blob/master/CHANGELOG.md#release-v1170---20240430).
+
+This release is a new release of v1.17 series.
+In this release, we added some new features for some plugins, and fixed bugs of Parser.
+
+## Enhancement
+
+### `in_tail`: Add `glob_policy` option for expanding glob capability of `path` and `exclude_path`
+
+In this release, we added a new option [glob_policy](https://docs.fluentd.org/input/tail#glob_policy) for [in_tail](https://docs.fluentd.org/input/tail) plugin.
+
+In previous versions, we can use only `*` in glob patterns for [path](https://docs.fluentd.org/input/tail#path) and [exclude_path](https://docs.fluentd.org/input/tail#exclude_path) option.
+
+Example:
+
+```
+path /path/to/*
+exclude_path ["/path/to/*.gz", "/path/to/*.zip"]
+```
+
+From this version, we can also use `[]`, `?` and `{}` in glob patterns depending on the `glob_policy` option.
```

Comment:
```suggestion
From this version, we can also use `[]`, `?`, and `{}` in glob patterns depending on the `glob_policy` option.
```

By contrast, comma should be needed here because already two or more items are enumerated.